hebrew/bar/beer/beer.md

```diff
@@ -0,0 +1,23 @@
+אתה הולך בדרכך לבר ועוברת לידך חבורה של 3 בנות חמודות שמצחקקות להן בהנאה ומחפשות איפה לבלות. הן פונות אליך ושואלות אם אתה מכיר בר טוב בסביבה.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.